package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@pact-foundation/pact",
   "version": "10.1.0",
   "description": "Pact for all things Javascript",
-  "main": "./src/index.js",
-  "types": "./src/index.d.ts",
+  "main": "./dist/src/index.js",
```

Comment:
This won't work in the current state - the publish script actually creates an empty directory that copies in the dist folder, so the old path was correct.

Probably the right move is to update the publish script so it doesn't do that anymore - it was only a hack so that both pact-js and pact-web could be published from this repo, and pact-web isn't published any more.